bliss/surveys/sdss_download.py

```diff
@@ -4,70 +4,98 @@
 
 import requests
 
+__all__ = [  # noqa: WPS410
```

Comment:
Better, I think, to keep all the sdss-specific code in `sdss.py`, rather than to spread it across two files

Reply:
Done.